docs/Splashkit/Extensions/MissingSwingameFunctionalities.md

```diff
@@ -0,0 +1,129 @@
+# Missing Functionalities from Swingame
+
+Splashkit is a recreation of an old game engine made by Swinburne University called Swingame, this 
+document aims to collect missing functions from [Swingame](https://github.com/macite/swingame) to 
+expand on Splashkit's functionality. Swingame's source code is predominently written in `Pascal` 
+so these functions will need to be translated before implementing them into Splashkit. 
+
+
+## Approach
+
+It is suggested that the missing functionalities are in relation to collision logic. By looking 
+into the `src` files within the 
+[Swingame repository](https://github.com/macite/swingame/tree/develop/CoreSDK/src) and comparing 
+them to that of the Splashkit `src` files, missing functionalities are found and documented.
+
+## To be added to Splashkit
+The following funtions and procedures are located within the 
+[src](https://github.com/macite/swingame/tree/develop/CoreSDK/src) folder of the Swingame Github 
+reposotiory, but cannot be located within the 
+[Splashkit src](https://github.com/splashkit/splashkit-core/tree/develop/coresdk/src/coresdk) 
+files. All descriptions are taken from their given depending Swingame`src` file.
```

Comment:
I am a little confused with this sentence? Are you trying to say something like, "All descriptions are taken from the associated SwinGame `src` file." Or something like this?

Reply:
That is what I meant, just couldn't word it just right.